RAOricutron/.gitmodules

```diff
@@ -0,0 +1,4 @@
+[submodule "RAIntegration"]
+	path = RAIntegration
+	url = https://github.com/rzumer/RAIntegration.git
+	branch = oric
```

Comment:
1) You should be using the RAInterface submodule
2) You should be using the top-level submodule that's available to all emulators.